src/mm/memory_manager.rs

```diff
@@ -240,18 +240,24 @@ pub fn is_live_object(object: ObjectReference) -> bool {
     object.is_live()
 }
 
-/// Is the object in the mapped memory?
+/// Is the object in the mapped memory? The runtime can use this function to check
+/// if an object is in MMTk heap.
 ///
 /// Arguments:
 /// * `object`: The object reference to query.
 pub fn is_mapped_object(object: ObjectReference) -> bool {
     object.is_mapped()
 }
 
-/// Is the address in the mapped memory?
+/// Is the address in the mapped memory? The runtime can use this function to check
+/// if an address is mapped by MMTk. Note that this is different than is_mapped_object().
+/// For malloc spaces, MMTk does not map those addresses (malloc does the mmap), so
+/// this function will return false, but is_mapped_object will may return true if the address
```

Comment:
change: "but is_mapped_object will may return true"
to
"but is_mapped_object() will return true"